src/skills-builder/skills-builder-modal/view-results/test/ViewResults.test.jsx

```diff
@@ -140,20 +152,42 @@ describe('view-results', () => {
           selected_recommendations: {
             job_id: 0,
             job_name: 'Prospector',
-            courserun_keys: [
-              {
-                title: 'Mining with the Mons',
-                courserun_key: 'MONS101',
-              },
-              {
-                title: 'The Art of Warren Upkeep',
-                courserun_key: 'WAR101',
-              },
-            ],
+            courserun_keys: {
+              course: [
+                {
+                  title: 'Mining with the Mons',
+                  courserun_key: 'MONS101',
+                },
+                {
+                  title: 'The Art of Warren Upkeep',
+                  courserun_key: 'WAR101',
+                },
+              ],
+            },
           },
         },
       );
     });
+
+    it('renders a list of recommendations for each line of business', () => {
+      windowSpy.mockImplementation(() => ({
+        location: {
+          origin: '/?product_types=2U_degree,boot_camp,executive_education,program,course&external_link=true',
+        },
+      }));
+
+      expect('"Prospector" degrees').toBeTruthy();
```

Comment:
Am I missing something about how this works? Aren't strings always going to be truthy?

Reply:
So does `expect()` automatically search through the context for a string? If you put any random string as the argument to expect, wouldn't it be truthy? I think you need to look for the string in the screen object.

Reply:
Oh yes! I didn't check the `screen`. I see what you mean now 👍  I'll fix this